### DISCUSSION TITLE
Contributions?

### DISCUSSION THREAD

**Discussion by brannonh** [dracula/dracula-theme#588] [General] [answered]
Hello!

I see quite a few theme contributions in Issues (including one of mine) that haven't been migrated to the @dracula org, going back two months. What is the normal process / timeline for this?

**Comment by bigpick** [comment, marked as answer]
Hey; I took a look through your issue (https://github.com/dracula/dracula-theme/issues/586) and thanks for making those updates; the timeline is not extremely rigid, since the inclusion of a new theme for promotion into the org is usually on Zeno's schedule, and they're a pretty busy person;) 

IMO, it's seems they take a pass ~1 a month to grab a set of themes off the issues list to be promoted into the org if they look ready; I started going through issue to try to get them cleaned up to reduce the number (as some are super old) but also so that when Zeno does have time to look it is as easy as possible for them (see: [ready-for-transfer-approval](https://github.com/dracula/dracula-theme/issues?q=is%3Aissue+is%3Aopen+label%3Aready-for-transfer-approval) label now)

**Reply by brannonh** [reply]
Thanks for the info and the work you do!
